test/Microsoft.ML.Tests/TrainerEstimators/MetalinearEstimators.cs

```diff
@@ -51,6 +51,36 @@ public void OVAUncalibrated()
             Done();
         }
 
+        /// <summary>
+        /// Test what OVA preserves key values for label.
+        /// </summary>
+        [Fact]
+        public void OvaKeyNames()
```

Comment:
Can we add this to `FunctionalTests` instead? Let's not keep adding tests that the product allows InternalsVisibleTo. That way we can test like a customer uses the product.